pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -41,20 +41,19 @@ import (
 
 const (
 	// ReasonCouldntCreateOrUpdateAffinityAssistantStatefulSetPerWorkspace indicates that a PipelineRun uses workspaces with PersistentVolumeClaim
-	// as a volume source and expect an Assistant StatefulSet, but couldn't create a StatefulSet.
+	// as a volume source and expect an Assistant StatefulSet in AffinityAssistantPerWorkspace behavior, but couldn't create a StatefulSet.
 	ReasonCouldntCreateOrUpdateAffinityAssistantStatefulSetPerWorkspace = "ReasonCouldntCreateOrUpdateAffinityAssistantStatefulSetPerWorkspace"
 
 	featureFlagDisableAffinityAssistantKey = "disable-affinity-assistant"
 )
 
-// createOrUpdateAffinityAssistantsPerAABehavior creates Affinity Assistant StatefulSets based on AffinityAssistantBehavior.
+// createOrUpdateAffinityAssistantsAndPVCsPerAABehavior creates Affinity Assistant StatefulSets and PVCs based on AffinityAssistantBehavior.
 // This is done to achieve Node Affinity for taskruns in a pipelinerun, and make it possible for the taskruns to execute parallel while sharing volume.
 // If the AffinityAssitantBehavior is AffinityAssistantPerWorkspace, it creates an Affinity Assistant for
 // every taskrun in the pipelinerun that use the same PVC based volume.
 // If the AffinityAssitantBehavior is AffinityAssistantPerPipelineRun or AffinityAssistantPerPipelineRunWithIsolation,
 // it creates one Affinity Assistant for the pipelinerun.
-// Other AffinityAssitantBehaviors are invalid.
-func (c *Reconciler) createOrUpdateAffinityAssistantsPerAABehavior(ctx context.Context, pr *v1.PipelineRun, aaBehavior aa.AffinityAssitantBehavior) error {
+func (c *Reconciler) createOrUpdateAffinityAssistantsAndPVCsPerAABehavior(ctx context.Context, pr *v1.PipelineRun, aaBehavior aa.AffinityAssitantBehavior) error {
```

Comment:
nit: `perAABehavior` is a bit redundant with the function arguments; we could shorten the function name to createOrUpdateAffinityAssistantsAndPVCs or `handleVolumeScheduling`

Reply:
Renamed to createOrUpdateAffinityAssistantsAndPVCs